src/main/kotlin/uk/gov/justice/digital/hmpps/hmppshdcapi/licences/PopulateLicenceDeletedAtMigration.kt

```diff
@@ -40,24 +41,26 @@ class PopulateLicenceDeletedAtMigration(
         batchSize = numberToMigrate,
         totalBatches = licencesRecords.totalPages,
         totalRemaining = licencesRecords.totalElements - licences.size,
+        lastIdProcessed = lastIdProcessed,
```

Comment:
I think that adding the lastIdProcessed to the Response will return it as part of the payload but want to check this.